services/app-api/handlers/reports/update.ts

```diff
@@ -39,89 +39,97 @@ export const updateReport = handler(async (event, context) => {
     if (getCurrentReport?.body) {
       const currentReport = JSON.parse(getCurrentReport.body);
       const { formTemplateId, fieldDataId } = currentReport;
+      if (formTemplateId && fieldDataId) {
+        // if report not in archived state, proceed with updates
+        if (!currentReport.archived) {
+          const state: string = event.pathParameters.state;
 
-      // if report not in archived state, proceed with updates
-      if (!currentReport.archived) {
-        const state: string = event.pathParameters.state;
+          // get formTemplate from s3 bucket (for passed fieldData validation)
+          const formTemplateParams = {
+            Bucket: process.env.MCPAR_FORM_BUCKET!,
+            Key: "formTemplates/" + state + "/" + formTemplateId,
+          };
+          const formTemplate: any = await s3Lib.get(formTemplateParams); // TODO: strict typing
 
-        // get formTemplate from s3 bucket (for passed fieldData validation)
-        const formTemplateParams = {
-          Bucket: process.env.MCPAR_FORM_BUCKET!,
-          Key: "formTemplates/" + state + "/" + formTemplateId,
-        };
-        const formTemplate: any = await s3Lib.get(formTemplateParams); // TODO: strict typing
+          // get existing fieldData from s3 bucket (for patching with passed data)
+          const fieldDataParams = {
+            Bucket: process.env.MCPAR_FORM_BUCKET!,
+            Key: "fieldDatas/" + state + "/" + fieldDataId,
```

Comment:
```suggestion
            Key: `fieldData/${state}/${fieldDataId}`,
```

typo here on `fieldData` (and maybe we change to literals for consistency with `create`